build-image/openhabian.conf

```diff
@@ -33,8 +33,7 @@ clonebranch=master
 mode=unattended
 
 # Java architecture mode
-# Use "32-bit" or "64-bit"
-java_arch="64-bit"
+# Use 32-bit or 64-bit
+java_arch=64-bit
```

Comment:
did you apply this to the other conf files as well?

Reply:
now I did